ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetWorkTest.java

```diff
@@ -108,9 +108,9 @@ public void shouldReturnCorrectResultOnHighBlockSeed() {
   }
 
   @Test
-  public void shouldReturnCorrectResultECIP1099CalculatorBeforeActivation() {
+  public void shouldReturnCorrectResultOnHighBlockSeedEcip1099() {
```

Comment:
block 60000 is actually in the low end of the new epoch.  I'd recommend 59_999 or 99_999, which are 0/1 and 1/3.